src/core/IO.pm6

```diff
@@ -9,20 +9,21 @@ enum SeekType (
   :SeekFromEnd(2),
 );
 enum ProtocolFamily (
-  :PF_LOCAL(0),
+  :PF_UNSPEC(0),
+  :PF_LOCAL(1),
   :PF_UNIX(1),
   :PF_INET(2),
-  :PF_INET6(3),
-  :PF_MAX(4),
+  :PF_INET6(10),
```

Comment:
This should be retrieved from the OS, Linux has `10`, Windows `23`, and BSD iirc `28`. how...

Reply:
We can adhere to the Linux numbers inside Rakudo, and transform those to platform specific numbers in the VM. but I really dislike that idea

Reply:
@zoffixznet you have any input on this?